tldextract/tests/main_test.py

```diff
@@ -36,29 +36,23 @@ def assert_extract(
         assert expected_domain == ext.domain
         assert expected_tld == ext.tld
 
-
```

Comment:
This violates PEP8 no? I couldn't figure out how to enforce it with Pylint though.